### PR TITLE
M3-4303 CMR: Filterable pills

### PR DESCRIPTION
**Pull request by Jskobos**
## Description

Allow filtering by clicking on the status pills above the Linodes table. Styling still in progress.

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx [hunk 1]
```diff
@@ -372,10 +384,24 @@ export class ListLinodes extends React.Component<CombinedProps, State> {
...
                                         })}
+                                        onClick={() =>
+                                          this.setFilterStatus('offline')
+                                        }
                                         label={`${linodesOfflineCount} OFFLINE`}
                                         {...chipProps}
                                       />
-                                    </Grid>
+                                      {filterStatus !== 'all' && (
+                                        <button
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:394]: Icon/Text a little off center here.

Screen Shot 2020-07-16 at 8 44 19 AM

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx [hunk 2]
```diff
@@ -247,20 +260,13 @@ export class ListLinodes extends React.Component<CombinedProps, State> {
...
+    const linodesBusyCount = filterLinodesByStatus('busy', linodesData).length;
 
-    const linodesOfflineCount = linodesData.filter(
-      linode => linode.status === 'offline' || linode.status === 'stopped'
-    ).length;
+    const linodesOfflineCount = filterLinodesByStatus('offline', linodesData)
+      .length;
 
     const chipProps = {
       component: 'span',
```
Comment (WilkinsKa1) [inline packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:272]: Updating these to be buttons/clickable should fix the issue John called out!

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx (hunk 1 above)
Comment (WilkinsKa1) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:394]: To fix the alignment, you could use IconTextLink instead.

**Comment by Jskobos** [comment]
@WilkinsKa1 @johnwcallahan feedback addressed. The alignment still looks a little weird, please let me know what you think Kayla.

**Pull request merged**